BeeSwift/RequestManager.swift

```diff
@@ -84,4 +84,9 @@ class RequestManager {
         return params
     }
 
+    class func addDatapoint(urtext: String, slug: String, success: ((Any?) -> Void)?, errorHandler: ((Error?, String?) -> Void)?) {
+        let params = ["urtext": urtext, "requestid": UUID().uuidString]
```

Comment:
I am not sure the benefit of sending the requestId when it is just some random string not being tracked by the app.